chain/client/src/client_actor.rs

```diff
@@ -1484,6 +1509,67 @@ impl ClientActor {
     }
 }
 
+struct StatePartsActor {
+    runtime: Arc<dyn RuntimeAdapter>,
+    client_addr: Addr<ClientActor>,
+}
+
+impl StatePartsActor {
+    fn apply_parts(
+        &mut self,
+        msg: &StatePartsMessage,
+    ) -> Result<(), near_chain_primitives::error::Error> {
+        let store = self.runtime.get_store();
```

Comment:
nit: since store is used at every request, it is better to make it a field in `StatePartsActor`, instead of getting it every time

Reply:
This is pretty heavy job, I do not see why we should optimize one Arc::clone out of it. But if for some reason in the future runtime.store will point somewhere else than it points during ClientActor initialization we will save someone a lot of debugging